local/compose/start.go

```diff
@@ -41,8 +49,31 @@ func (s *composeService) Start(ctx context.Context, project *types.Project, cons
 	if err != nil {
 		return err
 	}
-	if group != nil {
-		return group.Wait()
+
+	if options.Attach == nil {
+		return nil
+	}
+
+	eg, ctx := errgroup.WithContext(ctx)
+	for _, c := range containers {
+		c := c
+		eg.Go(func() error {
+			statusC, errC := s.apiClient.ContainerWait(ctx, c.ID, container.WaitConditionNotRunning)
+			select {
+			case status := <-statusC:
+				service := c.Labels[serviceLabel]
+				options.Attach.Status(service, getContainerNameWithoutProject(c), fmt.Sprintf("exited with code %d", status.StatusCode))
```

Comment:
We need to filter on which specific container we want to abort everything, or is this done in the listener somewhere ?

Reply:
I guess just still WIP

Reply:
`--abort-on-container-exit` as defined by docker-compose applies to **any** service to exit, not a specific one. This is a potential improvement to consider for compose future.

Reply:
indeed, I mixed this with `--exit-code-from`